src/jobs/helm-template.yaml

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
Why do we need this job?

Reply:
Because of the executor problem. Steps run in the same executor :/

So had to create a different job. Although this causes a different problem which I am looking at.

Reply:
So this is kind of sacrifice of DX for easiness of implementation? I don't like that. Would installing all the required tooling in architect docker image solve the problem?

Reply:
@kopiczko Yep, I agree it's not really 'elegant' and yeh it would be fixed by adding it to architect docker image (which I am going to do).

I am going to close this PR and reopen once changes are in the architect docker image